docs/auth/teletraan_auth.yaml

```diff
@@ -78,13 +78,14 @@ definitions:
           - GROUP
           - SYSTEM
           - PLACEMENTS
```

Comment:
Where is `PLACEMENTS` used? I wasn't able to find it from a quick search

Reply:
It's not used at the moment, and we might use it in the future. This is a list of allowed values.